_rules/iframe-non-empty-accessible-name-cae760.md

```diff
@@ -131,17 +132,11 @@ This `iframe` is not [included in the accessibility tree][] because of setting a
 <iframe style="display:none;" src="/test-assets/SC4-1-2-frame-doc.html"></iframe>
 ```
 
-#### Inapplicable Example 3
-
-This `iframe` element has a negative `tabindex` and therefore is not included in the [sequential focus navigation][].
```

Comment:
@dd8 per your suggestion, instead of moving this to failed example, I removed it, as it was similar to failed example 1 if moved to failed category. Thanks.

Reply:
Yep, I think it should be added as a fail, otherwise implementations that only test sequential focus navigation will show as correct and complete for all the tests.